doc/nrf/known_issues.rst

```diff
@@ -518,21 +484,206 @@ Closing sockets
   Moreover, closing a socket does not properly clean up all memory resources.
   If a socket is opened and closed multiple times, this  might starve the system.
 
+Multiprotocol Service Layer (MPSL)
+==================================
+
+The following known issues and limitations have been moved from :ref:`MPSL's changelog <nrfxlib:mpsl_changelog>` for the |NCS| v1.4.0 release, and can also affect some of the previous versions of |NCS|.
+
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
+
+DRGN-6362: Do not use the synthesized low frequency clock source
+  The synthesized low frequency clock source is neither tested nor intended for usage with MPSL.
+
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
+
+DRGN-14153: Radio Notification power performance penalty
+  The Radio Notification feature has a power performance penalty proportional to the notification distance.
+  This means an additional average current consumption of about 600 �A for the duration of the radio notification.
+
+.. rst-class:: v1-4-0
+
+Front-End Modules API only for 802.15.4
+  The FEM API is currently implemented in the 802.15.4 protocol only.
```

Comment:
Currently FEM API is not implemented for any protocol.

Reply:
Rephrased as per Teams f2f.